corehq/apps/registration/templates/registration/create_new_user.html

```diff
@@ -18,31 +17,12 @@
             </div>
         {% endfor %}
         <fieldset>
-            {% for field in form.visible_fields %}
-                <div class="control-group{% if field.errors %} error{% endif %}">
-                    <label class="control-label" for="{{ field.id }}">{{ field.label }}</label>
-                    <div class="controls">
-                        {% if field|is_checkbox %}
-                            <label class="checkbox">
-                                {{ field }} {{ field.help_text }}
-                            </label>
-                            {% for error in field.errors %}
-                                <span class="help-inline">{{ error }}</span>
-                            {% endfor %}
-                        {% else %}
-                            {{ field }}
-                            {% for error in field.errors %}
-                                <span class="help-inline">{{ error }}</span>
-                            {% endfor %}
-                            {% if field.help_text %}
-                                <p class="help-block">
-                                    {{ field.help_text }}
-                                </p>
-                            {% endif %}
-                        {% endif %}
-                    </div>
-                </div>
-            {% endfor %}
+            {% include "registration/partials/field.html" with field=form.full_name %}
```

Comment:
form.visible_fields sorts superclass's fields (project name) to the top, so I'm now ordering fields manually. Long-term, this form should probably converted to crispy and take advantage of the layout options that provides.